site/src/components/form.md

```diff
@@ -0,0 +1,130 @@
+---
+title: Form
+description: The form related components in the Atlas Design System
+template: standard
+---
+
+# Form
+
+Forms contain form control elements such as the following:
+
+- [`.button`](./button.md)
+- [`.checkbox`](./checkbox.md)
+- [`.help`](./help.md)
+- [`.input`](./input.md)
+- [`.label`](./label.md)
+- [`.radio`](./radio.md)
+- [`.select`](./select.md)
+- [`.textarea`](./textarea.md)
+- custom form elements
+
+## Form Field
+
+The `.field` component is a container for form control elements to provide consistency.
+It can be used to hold `.field-body` containers, which holds the individual elements.
+`field-label` is used for the element's label.
+
+```html
+<fieldset class="field border-none">
```

Comment:
lets remove border-none and move this over from docs-ui
https://dev.azure.com/ceapex/Engineering/_git/docs-ui?path=/packages/styles/src/base/_generic.scss&version=GBdevelop&_a=contents
```
fieldset {
	border: none;
	min-width: 0; // as a replaced element min-width:0 is required to enable it to work in responsive layouts
}
```

Reply:
Added to bare-elements